object-manipulation-3/main.js

```diff
@@ -0,0 +1,75 @@
+console.log('Lodash is loaded:', typeof _ !== 'undefined');
+var cards = [2, 3, 4, 5, 6, 7, 8, 9, 10, 10, 10, 10, 11];
+var p1 = {
+  name: 1,
+  hand: []
+};
+var p2 = {
+  name: 2,
+  hand: []
+};
+var p3 = {
+  name: 3,
+  hand: []
+};
+var p4 = {
+  name: 4,
+  hand: []
+};
+var deck = [];
+for (var i = 0; i < cards.length; i++) {
+  var diamond = {
+    rank: cards[i],
+    suit: 'diamond'
+  };
+  var club = {
+    rank: cards[i],
+    suit: 'club'
+  };
+  var heart = {
+    rank: cards[i],
+    suit: 'heart'
+  };
+  var spade = {
+    rank: cards[i],
+    suit: 'spade'
+  };
+  deck.push(diamond, club, heart, spade);
+}
+
+shuffleArray(deck);
+p1.hand.push(deck[0], deck[1]);
+p2.hand.push(deck[2], deck[3]);
+p3.hand.push(deck[4], deck[5]);
+p4.hand.push(deck[6], deck[7]);
```

Comment:
You should do this with a loop over the players, where you add two cards to each of their hands. Also, make sure you removed the cards from the deck once they are given to a player.